openedx/core/djangoapps/coursetalk/helpers.py

```diff
@@ -0,0 +1,34 @@
+"""
+CourseTalk widget helpers
+"""
+from __future__ import unicode_literals
+from openedx.core.djangoapps.coursetalk import models
+
+
+def get_coursetalk_course_key(course_key):
+    """
+    Return course key for coursetalk widget
+
+    CourseTalk unique key for a course contains only organization and course code.
+    :param course_key: SlashSeparatedCourseKey instance
+    :type course_key: SlashSeparatedCourseKey
+    :return: CourseTalk course key
+    :rtype: str
+    """
+    return '{0.org}_{0.course}'.format(course_key)
+
+
+def update_view_context(context, course_key):
```

Comment:
Could you rename this to incorporate `coursetalk` in some way? Simply updating a view context is very vague -- I'd rather the function be a little more self-documenting.